internal/website/blog/2022-08-31-sprint-17-update.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Sprint 17 Update
+description: Updates from the teams work on Sprint 17
+slug: sprint-17-update
+hide_table_of_contents: true
+---
+
+We finished Sprint 17 yesterday, our 17th 2-week Sprint of the year. Our current focus has been two-fold: first, to release a new version of Faust using Apollo as our GraphQL library and, second, to build a plan for how to best integrate Gutenberg (the block editor) into a headless site with the minimum amount of developer friction. As far as progress on both those endeavors is concerned, the Sprint was a success.
+
+## What we accomplished
+
+One of our more important features in Faust has been the support of Sitemaps as implemented by plugins such as [Yoast SEO](https://wordpress.org/plugins/wordpress-seo/) and others. While we've had the feature for a while, Next.js 12.2 [changed their middleware implementation rather significantly](https://nextjs.org/blog/next-12-2) leaving the feature broken for our users. We now have a fix in place that avoids using Next's middleware and hope to have it in a your hands in the coming weeks.
+
+In addition to updating our sitemap feature, the team has also been learning Gutenberg development to better understand WordPress development as a whole. Most of our backgrounds are either in classic WordPress or in JavaScript so Gutenberg, at least at the level we need to understand it, has been a learning experience for everyone this quarter. With the conclusion of Sprint 17 we feel we have enough experience with it to finally start looking at how we can use it to make headless development better. For example, what if you could re-use your components in both the editor and on your headless front-end? There are many questions and many avenues to pursue and, over the coming weeks, we'll be putting together a plan to do so. To me, personally, this is one of the most exciting possibilities for headless WordPress. Unlocking the potential of Gutenberg for headless sites will all for use and re-use of content on a scale that just isn't possible in class WordPress and the potential is amazing.
```

Comment:
This is reading so well! Great job, Chris! I love that we are regularly updating folks in such a conversational way.

My one suggestion (that you can take or leave), is to augment the second sentence to add clarity by adding a comma after JavaScript, and the next sentence fragment between hyphens instead.

Like so:
> either in classic WordPress or in JavaScript, so Gutenberg -at least at the level we need to understand it-

Reply:
I've added the comma, but not the hyphens. A long time ago a writing tutor convinced me to stop over-using hyphens and I just can't bring myself to make that change here :D

Reply:
Hahaha! Fair!